### PR TITLE
Fix astring import bug

### PR DESCRIPTION
**Pull request by benjamin-hodgson**
* astring is an `__esModule`-type module with no `default` export
* TypeScript emits code roughly equivalent to `require('astring').default` when the required module is an `__esModule`
* I encountered the bug when trying to run the NPM-distributed version of the plugin in my code
* I don't know why TypeScript allowed me to write `import astring` without an error, and I don't know why it behaves correctly in the test harness but the compiled code is wrong.
    * `astring`'s `d.ts` file uses ESM syntax, so TypeScript should know that there's no default export?
    * I do get a type error if I disable `esModuleInterop`.

Steps to reproduce:

* Install v1.4.0 from NPM and try to run it
* I get an exception from `stringifyAst` ("cannot read properties of undefined").
    * The generated JS code says `astring_1.default.generate(ast, astringOptions)`.
    * With this fix it says `astring.generate(ast, astringOptions);` (no `default`).
    * It was a bit tricky to track down the exception, because it happens in the Webpack pipeline. I had to run in the debugger to find it. Your mileage may vary

### CODE REVIEW

File: src/utils/stringify-ast.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import astring from 'astring';
+import * as astring from 'astring';
```
Comment (privatenumber) [inline src/utils/stringify-ast.ts:1]: Was it possible to just import `generate` since that's all we're using?

**Comment by privatenumber** [comment]
Seems this isn't happening with the tests because the tests are transpiled on-demand via esbuild instead of typescript

**Comment by privatenumber** [comment]
Thanks for the fix! Released in 1.4.1

**Comment by benjamin-hodgson** [comment]
Thank you!

**Pull request merged**